src/Form/FieldInput.js

```diff
@@ -17,6 +17,20 @@ class FieldInput extends React.Component {
     }
   }
 
+  componentDidMount() {
+    let inputElement = ReactDOM.findDOMNode(this.refs.inputElement);
+    if (inputElement != null &&
+        inputElement.type === 'text' &&
+        this.props.focused) {
+      // Don't interfere with existing transitions
```

Comment:
This is fine for now, but I'm sure we'll run into this problem eventually. So we'll have to fix it then.